### DISCUSSION TITLE
METplus 6 Beta 6 is not retaining my GRID_STAT_REGRID_TO_GRID setting

### DISCUSSION THREAD

**Discussion by lindsayblank** [dtcenter/METplus#2790] [Incoming] [answered]
Hello wonderful METplus folks,

I am working on upgrading METplus workflows from METplus 5.1.0 to METplus 6.0.0. I am using METplus 6.0.0 beta 6 and MET 12.0.0 beta 6 on Hera. 

In order to upgrade to METplus 6, I took the settings that differed from our custom GridStat config file and the default GridStat wrapped config file and set them in the METplus config file:
<img width="1649" alt="Screenshot 2024-11-13 at 4 02 45 PM" src="https://github.com/user-attachments/assets/9e59492e-6aab-4e5a-b7ea-f2d118dc05db">

I am running the workflow via rocoto. I got an error that my grids didn't match and I needed to select a grid to regrid to, which is odd, since I set that to FCST. When looking at the final METplus config, the GRID_STAT_REGRID_TO_GRID setting changes to NONE, but all other options are as I specified.
<img width="1643" alt="Screenshot 2024-11-13 at 4 04 59 PM" src="https://github.com/user-attachments/assets/ff1fecea-f62d-4d41-97ff-93da43741814">

Do you have any idea why the setting is being reverted?  Please let me know if I can provide any more information.

Thanks!
Lindsay

**Comment by CPKalb** [comment]
Hi Lindsay:

Would you be able to send over your configuration file so we can take a closer look.  You could post it here or point me to its location on Hera if that is easier.

Thanks,
Tina

**Comment by georgemccabe** [comment, marked as answer]
Hi Lindsay @lindsayblank,

I'm not able to recreate this issue locally. It may be a configuration issue. Perhaps you have `GRID_STAT_REGRID_TO_GRID = NONE` defined elsewhere in your configuration file(s)? If a variable is defined multiple times, the last instance of that variable is used. A quick test to see if that is the case is to move all of the new variables you defined to a config file that is passed to `run_metplus.py` last, or put them at the bottom of the configuration file.

If you are still having issues, could you send me the `run_metplus.py` command that you use with all of the argument, as well as all of the configuration files that you included in that command?

Thanks!

**Reply by lindsayblank** [reply]
George, you nailed it! There was an older setting further down that was overwriting the setting I put in. Thank you both for responding! Mark this as user error :) I'm rusty!

**Reply by georgemccabe** [reply]
Great to hear it was a simple fix! I'm going to close/lock this discussion but don't hesitate to create another discussion if you run into any other issues.
